src/ash/looprobot_choice.ash

```diff
@@ -22,6 +22,16 @@ void main(int choice, string page)
     else
       run_choice(2); // Decline
   }
+  else if (choice == 182) {
```

Comment:
Be nice if we had some comments on this to tell us what choices we think we're running without needing to run to the wiki, and what action the choice just did

Reply:
Think I solved this in the choice adventure comments and rework,